### ISSUE TITLE
Battery voltage sensing

### ISSUE THREAD

**Issue by misan** [cinderblock/3-Phase-Driver#3]
It appears that your voltage divider (two 10Kohm resistors) on the battery (12V) side can provide a voltage too high for the 5V 32u4 part. Not sure whether it is a typo on the schematic or that you trust that clamp diode on the controller input pin will take care of that, but it might be a problem.

**Comment by cinderblock** [comment]
You are correct, that is wrong. Thanks for pointing it out. It is a result of lazy design. It was one of the things to "figure out" during assembly as exact drive voltage had not been picked during design.

I'd like to start cleaning those little things up. Especially since I'm getting started on V2. Cheers!

**Comment by mkeyno** [comment]
@cinderblock are you still working on V2?

**Comment by mkeyno** [comment]
@cinderblock are you still working on V2?

**Comment by cinderblock** [comment]
@mkeyno I'm on V3~V5 depending on how you count! Some versions of this that I've worked on are unfortunately not going to be made public. However two new versions are published. Checkout the other two branches.

The "Hoverboard" branch is designed to fit the motor controller inside of the common "hoverboard" motors. 2 power wires and 1 USB cable going in gets you a full featured powerful servo motor. The "Quanum" branch is designed to fit onto the back of a specific motor from HobbyKing and interface with a different robot I'm making that we hope to publish soon.

**Comment by mkeyno** [comment]
@cinderblock,   I was  wondering whether on open source version there is more information beside the schematic  such as coding or pcb design

**Comment by cinderblock** [comment]
@mkeyno, the source files for the PCB are included. Unfortunately the software I use does not include a free viewer, nor is it easy to convert. I have yet to really try switching to a more open software but I'd like to eventually.

The gerbers are also published.

For software, checkout my other repository: https://github.com/cinderblock/3-Phase-Controller

We'd love to get this project more... polished... and easy for others to copy, replicate, and improve. Our goal was always just to make BLDC drivers/controllers easier and more accessible. Step 1 is just to get a system working. Step 2 is use it in many places to ensure reliability. Step 3 is to document and improve. If we can help you (or anyone) push this project forward, we're more than happy to help, time permitting.
